### PR TITLE
update link for teams

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Updates introduction link to use teams rather than channels.

#### Ticket Link
  Fixes https://github.com/mattermost/focalboard/issues/2709

### CODE REVIEW

File: mattermost-plugin/webapp/src/index.tsx [hunk 1]
```diff
@@ -236,8 +236,8 @@ export default class Plugin {
         } else {
             windowAny.frontendBaseURL = subpath + '/plug/focalboard'
             this.channelHeaderButtonId = registry.registerChannelHeaderButtonAction(<FocalboardIcon/>, () => {
-                const currentChannel = mmStore.getState().entities.channels.currentChannelId
-                window.open(`${window.location.origin}/plug/focalboard/workspace/${currentChannel}`)
+                const currentTeam = mmStore.getState().entities.teams.currentTeamId
+                window.open(`${window.location.origin}/plug/focalboard/team/${currentTeam}`)
```
Comment (Pinjasaur) [inline mattermost-plugin/webapp/src/index.tsx:240]: Any reason not doing the `_blank` and `noopener` like above here too?

```suggestion
                window.open(`${window.location.origin}/plug/focalboard/team/${currentTeam}`, '_blank', 'noopener')
```

Comment (sbishel) [reply mattermost-plugin/webapp/src/index.tsx:240]: After spending some time investigating, I'm not sure the "else" section even gets hit. The path provided `/plug/focalboard` is wrong as well, leading me to believe the section is unnecessary.

**Review by Pinjasaur — APPROVED** [review]
LGTM

File: mattermost-plugin/webapp/src/index.tsx [hunk 2]
```diff
@@ -233,13 +233,6 @@ export default class Plugin {
             }
 
             this.registry.registerPostWillRenderEmbedComponent((embed) => embed.type === 'boards', BoardsUnfurl, false)
-        } else {
-            windowAny.frontendBaseURL = subpath + '/plug/focalboard'
-            this.channelHeaderButtonId = registry.registerChannelHeaderButtonAction(<FocalboardIcon/>, () => {
-                const currentChannel = mmStore.getState().entities.channels.currentChannelId
-                window.open(`${window.location.origin}/plug/focalboard/workspace/${currentChannel}`)
-            }, 'Boards', 'Boards')
-            this.registry.registerCustomRoute('/', MainApp)
```
Comment (sbishel) [inline mattermost-plugin/webapp/src/index.tsx:242]: This section is never called. The initialize function is called by the web app - here 
https://github.com/mattermost/mattermost-webapp/blob/master/plugins/index.js#L172

The PluginRegistry is created with a "registerProduct" function. I'm leaving the check as a sanity check, but the else section is not needed and is wrong as well.

**Review by Pinjasaur — APPROVED** [review]

**Pull request merged**